src/act.other.cpp

```diff
@@ -4637,22 +4637,22 @@ ACMD(do_syspoints) {
     }
 
     // Mail the victim.
-    snprintf(buf, sizeof(buf), "You have been %s %d system points for %s.\r\n",
+    snprintf(buf, sizeof(buf), "You have been %s %d system points for%s\r\n",
             (award_mode ? "awarded" : "penalized"),
             k,
             reason);
     store_mail(idnum, ch, buf);
 
     // Notify the actor.
-    send_to_char(ch, "You %s %d system points %s %s for %s.\r\n",
+    send_to_char(ch, "You %s %d system points %s %s for %s\r\n",
```

Comment:
Same change re: terminal punctuation, it should be auto-supplied if the player has not supplied it already.